server/container/guice/jpa-guice/pom.xml

```diff
@@ -194,7 +194,7 @@
                  <dependency>
                     <groupId>org.apache.derby</groupId>
                     <artifactId>derby</artifactId>
-                    <scope>test</scope>
+                     <scope>compile</scope>
```

Comment:
I do not think this is correct. They would need to recompile JPA Guice with there driver added (so that it is part of the classpath).

Then they would need to add specific driver configuration, I guess in james-database.properties.
james-database